examples/tests/src/attach_file.test.ts

```diff
@@ -1,5 +1,5 @@
 import { afterEach, beforeEach, describe, expect, it } from "vitest";
-import { Buffer, PDFDocument } from "../../../dist/mupdfjs";
+import { PDFDocument } from "../../../dist/mupdfjs";
```

Comment:
Here is an example of where you include PDFDocument from mupdfjs, while it is included from mupdf in the next commit.